libherokubuildpack/Cargo.toml

```diff
@@ -45,6 +49,14 @@ termcolor = { version = "1.4.0", optional = true }
 thiserror = { version = "1.0.50", optional = true }
 toml = { workspace = true, optional = true }
 ureq = { version = "2.9.1", default-features = false, features = ["tls"], optional = true }
+lazy_static = { version = "1", optional = true }
```

Comment:
This dependency can be removed once the regex usage in `libherokubuildpack/src/output/util.rs` is switched to a non-regex approach.